cmd/fdsn-quake-consumer/assets/sc3ml_0.11__quakeml_1.2.xsl

```diff
@@ -178,18 +178,28 @@
  *
  *  * 10.12.2018: Put the non-QuakeML nodes in a custom namespace
  *
+ *  * 04.04.2022:
+ *    - Skip originUncertaintyDescription if value is set to
+ *      'probability density function' not supported by QuakeML.
+ *
+ *  * 31.10.2022: Improve performance when processing origins with many arrivals.
+ *
+ *  * 24.03.2023:
+ *    - Do not export duplicated picks referenced by different amplitudes.
+ *
  ********************************************************************** -->
 <xsl:stylesheet version="1.0"
         xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
         xmlns:scs="http://geofon.gfz-potsdam.de/ns/seiscomp3-schema/0.11"
         xmlns:qml="http://quakeml.org/xmlns/quakeml/1.0"
+        xmlns="http://quakeml.org/xmlns/bed/1.2"
         xmlns:q="http://quakeml.org/xmlns/quakeml/1.2"
         exclude-result-prefixes="scs qml xsl">
-    <xsl:output method="xml" encoding="UTF-8" indent="no" omit-xml-declaration="yes"/>
+    <xsl:output method="xml" encoding="UTF-8" indent="yes"/>
     <xsl:strip-space elements="*"/>
 
     <!-- Define parameters-->
-    <xsl:variable name="ID_PREFIX" select="'smi:nz.org.geonet/'"/>
+    <xsl:param name="ID_PREFIX" select="'smi:org.gfz-potsdam.de/geofon/'"/>
```

Comment:
@junghao We ll have to amend this part 
as per instructions:
```
*
 * This stylesheet converts a SC3ML to a QuakeML document. It may be invoked
 * e.g. using xalan or xsltproc:
 *
 *   xalan -in sc3ml.xml -xsl sc3ml_0.11__quakeml_1.2.xsl -out quakeml.xml
 *   xsltproc -o quakeml.xml sc3ml_0.11__quakeml_1.2.xsl sc3ml.xml
 *
 * You can also modify the default ID prefix with the reverse DNS name of your
 * institute by setting the ID_PREFIX param:
 *
 *   xalan -param ID_PREFIX "'smi:org.gfz-potsdam.de/geofon/'" -in sc3ml.xml -xsl sc3ml_0.11__quakeml_1.2.xsl -out quakeml.xml
 *   xsltproc -stringparam ID_PREFIX smi:org.gfz-potsdam.de/geofon/ -o quakeml.xml sc3ml_0.11__quakeml_1.2.xsl sc3ml.xml
```

This in order to get the Dat center righ in the quakeml in general 
this might need some work downstream with the script using it.